pkg/systemd/reporting.jsx

```diff
@@ -307,184 +319,124 @@ class BusWorkflowRow extends React.Component {
                             return;
                         }
 
-                        console.error(cockpit.format("reportd task for workflow $0 did not finish: $1", this.props.workflow[0], (ex.problem || ex.message)));
-                        this.setState({ message: _("Reporting failed") });
+                        console.error(cockpit.format("reportd task for workflow $0 did not finish: $1", workflow[0], (ex.problem || ex.message)));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19403-20230927-102042-c578ad22-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)